metadata-ingestion/src/datahub/ingestion/source/elastic_search.py

```diff
@@ -330,17 +339,25 @@ def _extract_mcps(self, index: str) -> Iterable[MetadataChangeProposalWrapper]:
         )
 
         # 4. Construct and emit properties if needed
+        properties: Dict[str, str] = {}
         index_aliases = raw_index_metadata.get("aliases", {}).keys()
-        if index_aliases:
-            yield MetadataChangeProposalWrapper(
-                entityType="dataset",
-                entityUrn=dataset_urn,
-                aspectName="datasetProperties",
-                aspect=DatasetPropertiesClass(
-                    customProperties={"aliases": ",".join(index_aliases)}
-                ),
-                changeType=ChangeTypeClass.UPSERT,
-            )
+        aliases: Dict[str, str] = {"aliases": ",".join(index_aliases)}
+        if len(index_aliases) > 0:
+            properties.update(aliases)
+        creation_date = float(index_properties["creation_date"])
+        creation_date_transform = time.strftime(
+            "%Y-%m-%d %H:%M:%S", time.localtime(creation_date / 1000)
+        )
+        index_properties["creation_date"] = creation_date_transform
+        properties.update(index_properties)
```

Comment:
Instead of lines 284-289, please use the following.
```python
properties.update({k:v for (k,v) in index_properties.items() if k not in ["analysis", "version", "lifecycle"]})
```